src_erl/erlBridge/nerlTests.erl

```diff
@@ -28,10 +28,8 @@ nerltest_print(String) ->
 
 test_envelope(Func, TestName, Rounds) ->
       nerltest_print(nerl:string_format("~p test starts for ~p rounds",[TestName, Rounds])),
-      Tic = nerl:tic(),
-      Func(Rounds),
-      {Diff, TimeUnit} = nerl:toc(Tic),
-      nerltest_print(nerl:string_format("Elapsed: ~p~p",[Diff,TimeUnit])), ok.
+      {TimeTookMicro, _RetVal} = timer:tc(?MODULE, Func, [Rounds]),
+      nerltest_print(nerl:string_format("Elapsed: ~p~p",[TimeTookMicro / 1000, ms])), ok.
```

Comment:
nerl_tools and nerl.erl are actually the same. 
Maybe we should merge them to a single file.

Reply:
However, I want us to be able to run tests of erlbridge without any dependency of compiling cowboy so if nerl_tools contain methods that use cowboy it is something that should be handled.